src/python/tests/test_orthology_benchmark.py

```diff
@@ -0,0 +1,224 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `orthology_benchmark.py` script.
+
+Typical usage example::
+
+    $ pytest test_orthology_benchmark.py
+
+"""
+
+from contextlib import nullcontext as does_not_raise
+from importlib.abc import Loader
+from importlib.machinery import ModuleSpec
+from importlib.util import module_from_spec, spec_from_file_location
+import os
+from pathlib import Path
+import sys
+from typing import ContextManager, Dict, List
+
+import sqlalchemy
+
+import pytest
+from pytest import raises
```

Comment:
```suggestion
from pytest import FixtureRequest, raises
```
(see following comment in `setup()`)

Reply:
I see in my current (local) code `from _pytest.fixtures import FixtureRequest` which I C/P-ed from our code base. What's the difference?

Reply:
It seems to be an update in `pytest` v6, which looks nicer, IMO (but doesn't really matter).

Reply:
Aha, I see. I also prefer less convoluted solutions so will go with the suggested one 🙂